docker-compose.yaml

```diff
@@ -56,6 +56,8 @@ services:
     env_file:
       - ./docker/pulp/env
     command: ['run', 'worker']
+    devices:
```

Comment:
I don't think we have access to `/dev/fuse` in OpenShift environment. Can you please elaborate?

Reply:
This is a buildah requirement for building so if it's not available in OpenShift then it won't run there. We need to figure out if OpenShift doesn't have this due to it being too new or if it's omitted by design. Who from that group can help us answer that?